enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/utils.ts

```diff
@@ -0,0 +1,97 @@
+import dayjs from "dayjs";
+import { c, t } from "ttag";
+
+import type { StaleCollectionItem } from "metabase-types/api";
+
+export const itemKeyFn = (item: StaleCollectionItem) =>
+  `${item.id}:${item.model}`;
+
+export const dateFilterOptions = [
+  {
+    label: c(`Not used in over -1 day`).t`-1 day`,
+    value: "one-day-back",
+    duration: [[-1, "day"]],
+  },
+  {
+    label: c(`Not used in over 1 month`).t`1 month`,
```

Comment:
I'm not sure the translator will understand what this context string means, without any further context. Maybe something like this could work: `c("Occurs in the phrase 'Not used in over 1 month'")`

Reply:
I realize this context string literally provides the linguistic context (that is, the very sentence fragment) in which the phrase occurs. But when it is seen in POEditor, without any further explanation, that might still be confusing.